doc/release-notes.rst

```diff
@@ -2,6 +2,135 @@
  Release Notes
 ===============
 
+v12.2.2 Luminous
+================
+
+This is the second bugfix release of Luminous v12.2.x long term stable release
+series. It contains a range of bug fixes and a few features across Bluestore,
+CephFS, RBD & RGW. We recommend all the users of 12.2.x series update.
+
+For more detailed information, see :download:`the complete changelog <changelog/v12.2.2.txt>`.
+
+
+Notable Changes
+---------------
+
+* Standby ceph-mgr daemons now redirect requests to the active messenger, easing configuration of tools and humans that access the web dashboard, restful API, or other ceph-mgr module services.
```

Comment:
Two things with this bullet point:

1. the line is very long - can it be wrapped/indented to fit into 80 characters like the lines in the intro above?
2. the formulation "easing configuration of tools and humans . . ." strikes me as odd? Maybe just drop "and humans"?